react/Tile/styles.styl

```diff
@@ -0,0 +1,45 @@
+$tileSize = 8.75rem
+$tileMobileHeight = 3.75rem
+
+.Tile
+  box-sizing border-box
+  position relative
+  display flex
+  flex-direction column
+  flex $tileSize 0 0
+  height $tileSize
+  align-items center
+  background var(--white)
+  border-radius 4px
+  border solid 1px var(--silver)
+  padding .375rem
+  margin-bottom 1rem
+  margin-right .75rem
+  overflow hidden
+  transition .15s ease all
+
+.Tile:hover,
+.Tile:active,
+.Tile:focus
+  box-shadow 0 4px 12px 0 #00000020
+  cursor pointer
+  transform scale(1.1)
+
+@media (min-width 1041px)
```

Comment:
Is it a new media query? I knew 1023 ( or 1200 but not 1041). Also still px

Reply:
I removed this small tweak since this breakpoint is not standard.